### ISSUE TITLE
Multiply or Divide Money with Money?

### ISSUE THREAD

**Issue by andrewtimberlake** [liuggio/money#9]
Does it make any sense to multiply or divide a `Money` with a `Money`?
I don’t usually say $5 * $5, it would be $5 * 5
I also don’t say $5 / $5 but rather $5 / 5

**Comment by andrewtimberlake** [comment]
A Money implementation I have in Ruby actually does the following:
```ruby
Money.new(10) / 2
# => [#<Money $5.00>, #<Money $5.00>]
```
While that initially seemed weird to me, it actually makes sense with how money is used.
```ruby
Money.new(11) / 3
# => [#<Money $3.67>, #<Money $3.67>, #<Money $3.66>]
```

**Comment by liuggio** [comment]
> Does it make any sense to multiply or divide a Money with a Money?

I think it doesn't... :+1: 

About the list as return value 
the other way is using upper and lower bound... doesn't look weird to me.

I think we should add also percentage with this notation, what do you think?

**Comment by andrewtimberlake** [comment]
Can you elaborate on "upper and lower bound"
Percentage is also a good idea. That would be useful for tax calculations. Any ideas on the interface?

**Comment by liuggio** [comment]
the other way was to pass options like:
**round-up** that rounds the money to the smallest integer greater than or equal to num
**round-down** that rounds the money to the largest integer less than or equal to num

but we could go with your idea... and let's think about it later...

**Comment by liuggio** [comment]
implemented by #22
